docs/feature_flags.md

```diff
@@ -239,9 +239,9 @@ Examples of equivalent durations:
 
 `--enable-feature=otlp-native-delta-ingestion`
 
-When enabled, allows for the native ingestion of delta OTLP metrics, storing the raw sample values without conversion. This cannot be enabled in conjunction with `otlp-deltatocumulative`.
+When enabled, allows for the native ingestion of delta OTLP metrics, storing the raw sample values without conversion. This cannot be enabled in conjunction with `otlp-deltatocumulative`, and `type-and-unit-labels` must be enabled.
```

Comment:
I believe we wanna recommend using the type-and-unit-labels feature gate for a more complete experience, but not demand